napari/layers/image/_image_constants.py

```diff
@@ -59,4 +59,5 @@ class Rendering(StringEnum):
     ADDITIVE = auto()
     ISO = auto()
     MIP = auto()
+    MINIP = auto()
```

Comment:
need to add to the docstring as well.
also feel like we need the names for max and min to seem more like obvious opposites now... @sofroniewn?

Reply:
MinIP MaxIP? will get on the docstring now

Reply:
I'm good with `MaxIP` and `MinIP`, but it might be nice to deprecate / alias `MIP` just so we don't immediately break others workflows if we do change it

Reply:
i'm also ok with leaving it `minip` and `mip` for now too

Reply:
is there no issue with inconsistency if we rename the `mip` from vispy? I've left it at `mip` and `minip` for now but happy to change whenever

Reply:
I vote for mip and minip. MIP is very well known as an acronym, and I think both backwards compatibility and consistency with existing vispy are concerns.